lib/yt/video_audit/base.rb

```diff
@@ -0,0 +1,19 @@
+module Yt
+  module VideoAudit
+    class Base
+      def initialize(options = {})
+        @videos = options[:videos]
+      end
+
+      # @return [Fixnum] number of all given videos.
+      def base_count
```

Comment:
I thought `total_count` can be confusing because it's an audit against a channel. But it's actually an audit for group of videos. I'll make change.